aptos-move/mvhashmap/src/versioned_code.rs

```diff
@@ -158,10 +159,10 @@ impl<K: Hash + Clone + Eq, V: TransactionWrite, X: Executable> VersionedCode<K,
                     Some(x) => Executable((x.clone(), ExecutableDescriptor::Published(hash))),
                     None => Module((module, hash)),
                 }),
-                Err(NotFound) => v
-                    .base_executable
-                    .as_ref()
-                    .map(|x| Executable((x.clone(), ExecutableDescriptor::Storage)))
+                Err(NotFound) => self
+                    .base_executables
```

Comment:
What if the key is not in values? We should check base_executables in this case as well, right?